app/templates/application.hbs

```diff
@@ -22,7 +22,7 @@
       {{/link-to}}
     </li>
     <li>
-      <a href="/explorer/cities/NYC?compareTo=1"
+      <a onclick={{action this.openExplorerURL}}
         class={{(if (eq this.router.currentRouteName "explorer") "active-route")}}
       >
```

Comment:
So here's something interesting – it looks like replacing this `a` tag with a `link-to` 'explorer' 'cities' 'NYC' causes the app to use the [queryParams we have bound](https://guides.emberjs.com/v3.4.0/routing/query-params/#toc_specifying-query-parameters) in the explorer [controller](https://github.com/NYCPlanning/labs-factfinder/blob/646f6bc3c9703e34b7f68797da1799b588e5d992/app/controllers/explorer.js#L17-L41). I'm willing to bet this is how the app was working previously. With this approach we don't have to use storage at all, but unfortunately the filters are only saved between the Map and Data Explorer tabs – hitting View Data will still lead users to the default Data Explorer view for that tract.

Reply:
cc @TylerMatteo @godfreyyeung

Reply:
Well that's promising, at least. I'm going to look into how the "View Data" button works because I'm pretty sure this isn't going to pass testing without that. I'm pretty sure users use that button much more than the actual "Data Explorer" tab.